lib/game.rb

```diff
@@ -0,0 +1,31 @@
+require 'CSV'
+class Game
+  @@all = []
+  attr_reader :game_id, :season,
+              :type, :date_time,
+              :away_team_id, :home_team_id,
+              :away_goals, :home_goals,
+              :venue, :venue_link
+
+  def self.from_csv(file_path)
+    csv = CSV.read("#{file_path}", headers: true, header_converters: :symbol )
+    @@all = csv.map{|row| Game.new(row)}
+  end
+
+  def self.all
```

Comment:
I like that we decided on the class method here and am excited to learn more about/work with it 🚀